docs/commerce/2.x/en/managing-a-catalog/creating-and-managing-products/channels/managing-channels.md

```diff
@@ -22,9 +22,9 @@ The channel is now created. Products may now be configured to be filtered by thi
 
 At this point, the channel is not yet been associated with any store site.
 
-## Associating a Channel with a Store Site
```

Comment:
In my mind, a Channel is a Channel, a Site is a Site, and a Store Site exists when a channel is associated with a site and therefore can be used as a store by Liferay Commerce. So the headline here should ready "Associating a Channel with a Site" or "...DXP Site" or "...Liferay Site." I don't have a strong opinion one way or the other.

Reply:
Going with _DXP site_ since _Liferay Site_ implies www.liferay.com